azkaban-db/src/main/sql/upgrade.3.86.0.to.3.87.0.sql

```diff
@@ -0,0 +1,6 @@
+-- DB Migration from release 3.86.0 to 3.87.0
```

Comment:
Current release is 3.104.3. Could you please rename the file to `upgrade.3.104.0.to.3.105.0.sql` and modify this line as well.